src/dms/tests/EdFi.DataManagementService.Tests.E2E/Features/General/DiscoveryAPI.feature

```diff
@@ -4,7 +4,7 @@ Feature: The Discovery API provides information about the application version, s
         Scenario: 01 GET / returns the root Discovery API document
              When a GET request is made to "/"
              Then it should respond with 200
-              And the general response body is
+              And the discovery API root response body is
```

Comment:
This was the original failing test due to a circular inside/outside docker issue